### DISCUSSION TITLE
Using Openroad within a conda environment

### DISCUSSION THREAD

**Discussion by Akilax0** [The-OpenROAD-Project/OpenROAD#5191] [Q&A] [answered]
Can I direct openroad -python interpreter to use the python version installed in a conda environment? I have all the dependencies to use Openroad & circuit Ops for ML Ops installed in this environment, so need openroad to detect those packages as well.

**Comment by maliberty** [comment]
You can't change the python interpreter as it is compiled in.  You can set the PYTHONPATH to make packages loadable.

**Reply by Akilax0** [reply]
Been trying to load as such but still uses the default installed python.  But won't use the conda installed one. 
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/46198141/2c252773-68e5-4a2d-a5ce-27a5a9878baa)


The conda installed python version is Python 3.12.3 
![image](https://github.com/The-OpenROAD-Project/OpenROAD/assets/46198141/1d431990-1e8d-48b9-a698-8cbbf68ca712)

**Reply by maliberty** [reply]
You are showing the python version which is not changeable. I was referring to package loading

**Reply by Akilax0** [reply, marked as answer]
Okay does that mean even though Python version is not changeable, we can have a conda environment with required packages installed with same python version to be used with PYTHONPATH

**Reply by maliberty** [reply]
I'm hoping so

**Reply by Akilax0** [reply]
This works! Thanks
